data/battle_anim_scripts.s

```diff
@@ -3911,9 +3911,9 @@ Move_LUSTER_PURGE: @ 81CC95B
 	createsprite gBattleAnimSpriteTemplate_83DA9E0, 41, 0, 0, 0, 0
 	delay 20
 	createvisualtask sub_80E2A7C, 5, 5, 2, 0, 16, -1
-	createvisualtask sub_80E2C60, 5, 10267, 2, 0, 16, -1
+	createvisualtask AnimTask_BlendSpriteColor, 5, 10267, 2, 0, 16, -1
 	waitforvisualfinish
-	createvisualtask sub_80E2C60, 5, 10135, 0, 12, 12, 23552
+	createvisualtask AnimTask_BlendSpriteColor, 5, 10135, 0, 12, 12, 23552
```

Comment:
The last argument of this `AnimTask_BlendSpriteColor` is an RGB color, so it can be updated to use the `rgb` macro, rather than raw integers, lik 23552